README.md

```diff
@@ -17,48 +17,23 @@ Official React Native bindings for Bitmovin's mobile Player SDKs.
 - [Bitmovin Player React Native](#bitmovin-player-react-native)
   - [Platform Support](#platform-support)
   - [Feature Support](#feature-support)
-  - [Installation](#installation)
-    - [Add package dependency](#add-package-dependency)
-    - [Setup iOS Player SDK](#setup-ios-player-sdk)
-    - [Setup Android Player SDK](#setup-android-player-sdk)
-  - [Getting Started](#getting-started)
-    - [Setting up a license key](#setting-up-a-license-key)
-      - [Through code](#through-code)
-      - [Through `Info.plist`](#through-infoplist)
-      - [Through `AndroidManifest.xml`](#through-androidmanifestxml)
-    - [Setting up the playback configuration](#setting-up-the-playback-configuration)
-    - [Accessing native `Player` instances](#accessing-native-player-instances)
-    - [Listening to events](#listening-to-events)
-    - [Enabling DRM protection](#enabling-drm-protection)
-      - [Prepare hooks](#prepare-hooks)
-    - [Adding external subtitle tracks](#adding-external-subtitle-tracks)
-    - [Adding external thumbnail track](#adding-external-thumbnail-track)
-    - [Enabling Picture in Picture mode](#enabling-picture-in-picture-mode)
-      - [Android](#android)
-      - [iOS](#ios)
-      - [Showing the Picture in Picture UI option](#showing-the-picture-in-picture-ui-option)
-      - [Supported Picture in Picture events](#supported-picture-in-picture-events)
-    - [Customize HTML UI](#customize-html-ui-android-and-ios-only)
-    - [Setting up fullscreen handling](#setting-up-fullscreen-handling)
-      - [Supported fullscreen related events](#supported-fullscreen-related-events)
-    - [Setting up ads](#setting-up-ads)
-      - [Static ads configuration](#static-ads-configuration)
-      - [Dynamic ads scheduling](#dynamic-ads-scheduling)
-      - [Supported ads events](#supported-ads-events)
-    - [Setting up analytics](#setting-up-analytics)
+  - [Documentation](#documentation)
+    - [Getting Started](#getting-started-guide)
+    - [Feature Guides](#feature-guides)
+    - [Sample Application](#sample-application)
   - [Contributing](#contributing)
 
 ## Platform Support
 
 This library requires at least React Native 0.64+ and React 17+ to work properly. The currently supported platforms are:
 
-- iOS 12.0+
-- tvOS 12.0+
-- Android API 16+
-- Android TV API 17+
-- Fire TV (just make sure the Android API level is at least 17+)
+- iOS 14.0+
+- tvOS 14.0+
+- Android API Level 16+ (note that DRM-protected streams require API Level 19+)
```

Comment:
Based on the project settings we require min SDK version 21 on Android

see: https://github.com/bitmovin/bitmovin-player-react-native/blob/development/android/build.gradle#LL33C1-L34C1

@zigavehovec to confirm

Reply:
Confirming. React native version `0.64` changed minSdkVersion version to 21 so it was not possible to compile with minSdkVersion set at 16.

Changelog: https://reactnative.dev/blog/2021/03/12/version-0.64#major-dependency-version-changes

Reply:
Thanks, this also applies to Android TV? @zigavehovec

Reply:
Yes, this also applies to Android TV.